completions/fish/crio.fish

```diff
@@ -161,6 +161,7 @@ complete -c crio -n '__fish_crio_no_subcommand' -f -l stream-port -r -d 'Bind po
 complete -c crio -n '__fish_crio_no_subcommand' -l stream-tls-ca -r -d 'Path to the x509 CA(s) file used to verify and authenticate client communication with the encrypted stream. This file can change and CRI-O will automatically pick up the changes within 5 minutes.'
 complete -c crio -n '__fish_crio_no_subcommand' -l stream-tls-cert -r -d 'Path to the x509 certificate file used to serve the encrypted stream. This file can change and CRI-O will automatically pick up the changes within 5 minutes.'
 complete -c crio -n '__fish_crio_no_subcommand' -l stream-tls-key -r -d 'Path to the key file used to serve the encrypted stream. This file can change and CRI-O will automatically pick up the changes within 5 minutes.'
+complete -c crio -n '__fish_crio_no_subcommand' -f -l time-zone -s tz -r -d 'Specifies the timezone configuration for CRI-O. If an empty string is provided, CRI-O will use the timezone of the host machine but displays it in UTC.'
```

Comment:
nit: time zone*

Reply:
@sohankunkerkar mentioned offline that this is the american spelling, which we have typically favored over the british.